### ISSUE TITLE
All bones of humans get crushed when their foot is crushed with a wall

### ISSUE THREAD

**Issue by ras09191** [studio-minus/ppg-bugs-and-requests#2868]
### Describe the bug

All bones of humans get broken when i drop a wall on their foot. Normally, only their foot should be broken, but instead, it breaks all bones of humans.

### Clearly describe how to reproduce the bug

1. Spawn in a human
2. Spawn a wall and make it drop on his foot
3. Watch as all the bones of the human break

### Mod list

Decals mod, humans+ mod, torture mod, the ultimate car mod, water physics mod, animated weapons+, taser, functioning armor, chemical warfare

### Game version

1.26.6

### Operating System

Windows 11

**Comment by Drogo-man** [comment]
Unity 👏 fuckery 👏

**Comment by ras09191** [comment]
Fuck Unity

**Comment by Drogo-man** [comment]
Yeah, I tried to get into game devving thru unity, and I just slammed my head into my desk and gave up 3 hours in.

**Comment by maplestake101** [comment]
this is kind of fun
